bindings/python/sample_x86.py

```diff
@@ -329,7 +329,7 @@ def test_i386_reg_save():
         mu.emu_start(address, address+1)
 
         print(">>> save the register state")
-        saved_regs = mu.save_regs()
+        saved_regs = mu.regs_save()
```

Comment:
i think the method name should be consistent with the API name. so instead of `regs_save`, how about `regstate_save`?